src/Findologic/Request/FindologicRequestFactory.php

```diff
@@ -74,6 +74,9 @@ protected function setDefaults(
         $searchNavigationRequest->setUserIp($this->fetchClientIp());
         $searchNavigationRequest->setRevision($this->getPluginVersion());
         $searchNavigationRequest->setOutputAdapter(OutputAdapter::XML_21);
+        $searchNavigationRequest->addIndividualParam("shopType", "Shopware6", true);
+        $searchNavigationRequest->addIndividualParam("shopVersion", Utils::getCleanShopwareVersion(), true);
```

Comment:
Use single quotes.

Reply:
@D3xter7 this is not resolved.

Reply:
What does the 3rd parameter do? Why have you passed `true` in here?

Reply:
> 

I have done this in the second last commit.

Reply:
> 

I didn't test that why it is required, but this function is working fine and required the third parameter as boolean.

Reply:
@D3xter7 the double quote is not fixed, please check again. When a comment as such is made, please make sure to check other places too where this is applicable as previously also mentioned to you by Dominik.